### ISSUE TITLE
New utility - Placeholder

### ISSUE THREAD

**Issue by afuno** [tailwindcss/tailwindcss#1060]
I suggest adding customization to placeholder. For example, setting the opacity and colors for the fields.

The default now is the transparency (opacity) of the color of the text. But this is not always good. Sometimes the text of the field is different from the placeholder color.

**Comment by brasilikum** [comment]
Given that probably not everyone needs something else than the default, have you looked into implementing this as a plugin?

**Comment by stancl** [comment]
This is something that can be done very easily with vanilla CSS.

```css
*::placeholder {
    color: red;
}
```

I don't think this should be a utility, just like `*::selected` shouldn't be a utility.

**Comment by brasilikum** [comment]
Support is incomplete (https://caniuse.com/#search=placeholder) but autoprefixer can help:
https://autoprefixer.github.io/

```css
*::-webkit-input-placeholder {
    color: red
}
*::-moz-placeholder {
    color: red
}
*:-ms-input-placeholder {
    color: red
}
*::-ms-input-placeholder {
    color: red
}
*::placeholder {
    color: red
}
```

I agree that this should not be part of core tailwind.

**Comment by adamwathan** [comment]
Implemented in #1063
